akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -467,6 +470,14 @@ object Multipart {
           FastFuture.successful(this)
         override def productPrefix = "FormData.BodyPart.Strict"
       }
+
+      /** Java API */
+      object StrictBuilder {
+        def apply(_name: String, _entity: HttpEntity.Strict,
```

Comment:
This API is not directly exposed, it is only used by `HttpEntities.java`. A bit of a hack, had to be done this way in order to allow use from Java.